src/commands/dev-env-sync-sql.js

```diff
@@ -58,12 +53,13 @@ async function extractSiteUrls( sqlFile ) {
 		readInterface.on( 'line', line => {
 			const domain = findSiteHomeUrl( line );
 			if ( domain ) {
-				domains.add( '//' + domain );
+				domains.add( domain );
 			}
 		} );
 
 		readInterface.on( 'close', () => {
-			resolve( Array.from( domains ) );
+			// Soring by length so that longest domains are replaced first
```

Comment:
We want the longest domains to be replaced first to handle issues like this:

Say, we do the following:
```
wpvip.com -> foo.com
static.wpvip.com -> bar.com
```

If we replace `wpvip.com` before the other one, we end up getting `static.foo.com`, and the `static.wpvip.com -> bar.com` never happens.